ext/scripts/install_scripts/install_via_pip.pl

```diff
@@ -32,13 +34,17 @@ =head1 SYNOPSIS
 my $with_versions = 0;
```

Comment:
I'm shocked to learn that we are now also using Perl.
.pl always has been associated with Prolog for me ... which I prefer.

(please ignore this comment!)

Reply:
I needed a scripting language which didn't interfere with the UDFs and is small and per default in the ubuntu